v2/src/modules/address/ModuleAddressNftTransfers.vue

```diff
@@ -0,0 +1,393 @@
+                            :subtext="`${tokenCount} total NFT's`"
+                        />
+                    </div>
+                    <span v-if="props.isOverview || mdAndDown" class="text-h6 font-weight-bold">NFT History</span>
+                    <app-new-update
+                        :icon-only="props.isOverview"
+                        text="New ERC20 Transfers"
+                        :update-count="props.newErc721Transfer"
+                        @reload="setPage(0, true)"
+                    />
+                </v-row>
+            </div>
+            <app-btn v-if="props.isOverview && !mdAndDown" text="More" isSmall icon="east" @click="goToNftTransfersPage"></app-btn>
+            <app-btn-icon v-if="props.isOverview && mdAndDown" icon="more_horiz" @click="goToNftTransfersPage"></app-btn-icon>
+        </v-card-title>
+        <div :class="!props.isOverview && !mdAndDown ? 'pt-13' : null">
+            <!--            Table Header-->
+
+            <v-row v-if="!mdAndDown" class="my-0 text-body-1 text-info">
+                <v-col :cols="props.isOverview ? 4 : 3" class="py-0"> Name/Id </v-col>
+                <v-col :cols="props.isOverview ? 2 : 1" class="py-0"> Copies </v-col>
+                <v-col :cols="props.isOverview ? 2 : 1" class="py-0"> From/To </v-col>
+                <v-col :cols="props.isOverview ? 4 : 3" class="py-0"> Address </v-col>
+                <v-col v-if="!props.isOverview" cols="2" class="py-0"> Hash </v-col>
+                <v-col v-if="!props.isOverview" cols="2" class="py-0"> Timestamp </v-col>
+            </v-row>
+            <v-divider class="my-0 mt-md-4 mx-n4 mx-sm-n6" />
+            <template v-if="initialLoad">
+                <div v-for="item in 10" :key="item" class="my-2">
+                    <div class="skeleton-box rounded-xl mt-1 my-4" style="height: 24px"></div>
+                </div>
+            </template>
+            <template v-else>
+                <template v-if="transfers.length < 1">
+                    <p class="text-h4 text-center my-2">This address does not hold any NFTs (ERC721 tokens)</p>
+                </template>
+                <template v-else>
+                    <div v-for="(transfer, index) in transfers" :key="`${transfer.transfer.transactionHash} - ${index}`" class="position-relative">
+                        <!--
+                                       ========================
+                                         Desktop View
+                                       =========================
+                                -->
+                        <v-row v-if="!mdAndDown" class="my-5 px-0 text-subtitle-2 font-weight-regular" align="center">
+                            <v-col :cols="props.isOverview ? 4 : 3" class="py-0">
+                                <v-row class="ma-0 flex-nowrap" align="center">
+                                    <img :src="getImg(transfer)" alt="" height="41" width="41" class="mr-2 rounded-circle" />
```

Comment:
transfer object can possible be null, please add a check,

Reply:
The `getImg` function has a check for it. Here: https://github.com/EthVM/EthVM/blob/f097ba80f91fb322fbda426a57c7bdb6335b8379/v2/src/modules/address/ModuleAddressNftTransfers.vue#L293-L299